src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -16,40 +16,37 @@ import PropTypes from 'prop-types'
 import shortid from 'shortid'
 const _ = require('lodash')
 
-export const valueTemplateRefTest = (property) => {
-  return Boolean(property.valueConstraint != null &&
-    property.valueConstraint.valueTemplateRefs != null &&
-    property.valueConstraint.valueTemplateRefs.length > 0)
+export const hasValueTemplateRef = (property) => {
```

Comment:
I struggled with a method with "Test" in the name in the source code -- hopefully this makes sense to all